src/sentry/static/sentry/app/components/activity/note/inputWithStorage.jsx

```diff
@@ -0,0 +1,126 @@
+import _ from 'lodash';
+import PropTypes from 'prop-types';
+import React from 'react';
+import * as Sentry from '@sentry/browser';
+
+import NoteInput from 'app/components/activity/note/input';
+import localStorage from 'app/utils/localStorage';
+
+class NoteInputWithStorage extends React.Component {
+  static propTypes = {
+    storageKey: PropTypes.string.isRequired,
+    itemKey: PropTypes.string.isRequired,
+    onLoad: PropTypes.func.isRequired,
+    onSave: PropTypes.func.isRequired,
+    onChange: PropTypes.func,
+    onCreate: PropTypes.func,
+  };
+
+  static defaultProps = {
+    onSave: i => i,
+    onLoad: i => i,
+  };
+
+  fetchFromStorage = () => {
+    const {storageKey} = this.props;
+
+    const storage = localStorage.getItem(storageKey);
+    if (!storage) {
+      return null;
+    }
+
+    try {
+      return JSON.parse(storage);
+    } catch (err) {
+      Sentry.captureException(err);
```

Comment:
Should we also capture the `storage` data, or will it be picked up automatically?